VideoApi/VideoApi.DAL/Mappings/TelephoneParticipantMap.cs

```diff
@@ -10,10 +10,11 @@ public class TelephoneParticipantMap : IEntityTypeConfiguration<TelephonePartici
     public void Configure(EntityTypeBuilder<TelephoneParticipant> builder)
     {
         builder.ToTable(nameof(TelephoneParticipant));
-            
+
         builder.HasKey(x => x.Id);
+        builder.Property(x => x.Id).ValueGeneratedNever();
         builder.Property(x => x.TelephoneNumber).IsRequired();
-        builder.Property(x => x.State).IsRequired().HasDefaultValue(TelephoneState.Connected);
-        builder.Property(x => x.CurrentRoom).IsRequired().HasDefaultValue(RoomType.WaitingRoom);
+        builder.Property(x => x.State).IsRequired();
+        builder.Property(x => x.CurrentRoom).IsRequired(false);
```

Comment:
is currentRoom not nullable by default? Does it need IsRequired(false)?

Reply:
The previous migration made it required, so I need to set it to false to be nullable